src/core/utils/url.js

```diff
@@ -23,11 +28,60 @@ export function removeUndefinedProps(object: Object): Object {
 export function addQueryParams(
   urlString: string,
   queryParams: { [key: string]: ?string | number } = {},
+  _config: typeof config = config,
 ): string {
+  let adjustedQueryParams = { ...queryParams };
```

Comment:
If this was python I'd really expect the default value to be defined in an `else:` rather than creating `adjustedQueryParams` and then overwriting it straight away inside the if but I guess this is "normal for javascript"...